LayoutTests/fast/css/resize-mutation-observer-expected.txt

```diff
@@ -0,0 +1,13 @@
+layer at (0,0) size 800x600
+  RenderView at (0,0) size 800x600
+layer at (0,0) size 800x204
+  RenderBlock {HTML} at (0,0) size 800x204
+    RenderBody {BODY} at (8,8) size 784x183
+      RenderBlock {PRE} at (0,138) size 784x45
+        RenderText {#text} at (0,0) size 250x45
+          text run at (0,0) width 250: "Style attribute change observed:"
+          text run at (249,0) width 1: " "
+          text run at (0,15) width 0: " "
+          text run at (0,30) width 180: "attributes, style, null"
+layer at (8,8) size 173x125 clip at (10,10) size 169x121
+  RenderBlock {DIV} at (0,0) size 173x125 [border: (2px solid #0000FF)]
```

Comment:
Please don't make new render tree dump tests. They are platform dependent and generally a pain. Instead use text dumps (`window.testRunner?.dumpAsText()`) or reftests.